mobile/src/main/java/org/hawkular/client/android/backend/BackendPipes.java

```diff
@@ -60,19 +60,19 @@ private Paths() {
         public static final String ALERT_ACKNOWLEDGE = "alerts/ack";
         public static final String ALERT_NOTE = "alerts/note";
         public static final String ALERT_RESOLVE = "alerts/resolve";
-        public static final String ENVIRONMENTS = "inventory/deprecated/environments";
-        public static final String FEEDS = "inventory/deprecated/feeds";
-        public static final String FEED_METRICS = "inventory/deprecated/feeds/%s/resources/%s/metrics";
-        public static final String FEED_RESOURCES = "inventory/deprecated/feeds/%s/resources";
-        public static final String FEED_REC_RESOURCES = "inventory/deprecated/feeds/%s/resources/%s/children";
-        public static final String METRICS = "inventory/deprecated/%s/resources/%s/metrics";
+        public static final String ENVIRONMENTS = "inventory/traversal/type=e";
+        public static final String FEEDS = "inventory/traversal/type=f";
+        public static final String FEED_METRICS = "inventory/traversal/f;{f}/{r}/rl;incorporates/type=m";
+        public static final String FEED_RESOURCES = "inventory/traversal/f;{f}/type=r";
+        public static final String FEED_CHILD_RESOURCES = "inventory/traversal/f;{f}/{r}/type=r";
+        public static final String METRICS = "inventory/traversal/e;{e}/{r}/type=m";
         public static final String METRIC_DATA_AVAILABILITY= "metrics/availability/%s/data";
         public static final String METRIC_DATA_COUNTER = "metrics/counters/%s/data";
         public static final String METRIC_DATA_GAUGE = "metrics/gauges/%s/data";
-        public static final String OPERATIONS = "inventory/deprecated/feeds/%s/resourceTypes/%s/operationTypes";
+        public static final String OPERATIONS = "inventory/traversal/f;{f}/rt;{rt}/type=ot";
         public static final String PERSONA = "hawkular/metrics/tenants";
         public static final String PERSONAS = "hawkular/metrics/tenants";
-        public static final String RESOURCES = "inventory/deprecated/%s/resources";
```

Comment:
earlier formatting of String was used but now it is using string replacements for {..?} So that everything can be handled inside CanonicalPath class of this client